packages/shared/src/components/HorizontalScroll/useCalculateVisibleElements.ts

```diff
@@ -31,7 +31,9 @@ export const useCalculateVisibleElements = <El extends HTMLElement>({
       return;
     }
 
-    setIsOverflowing(currentRef.scrollWidth > currentRef.clientWidth);
+    setTimeout(() => {
+      setIsOverflowing(currentRef.scrollWidth > currentRef.clientWidth);
+    }, 500);
```

Comment:
I don't like this very much so open for any other ideas...why this was added was because of the implementation on tags/sources, the scrollWidth takes a bit of time to actually update to the real value, meaning without the timeout, it won't show the buttons at all (it does if u refresh...) :/

Reply:
@dailydotdev/web-team